stacker/actions/build.py

```diff
@@ -290,22 +290,24 @@ def pre_run(self, outline=False, *args, **kwargs):
             util.handle_hooks('pre_build', pre_build, self.provider.region,
                               self.context)
 
-    def run(self, outline=False, tail=False, *args, **kwargs):
+    def run(self, outline=False, tail=False, dump=False, *args, **kwargs):
         """Kicks off the build/update of the stacks in the stack_definitions.
 
         This is the main entry point for the Builder.
 
         """
         plan = self._generate_plan(tail=tail)
-        if not outline:
+        if outline:
+            plan.outline()
+        elif dump:
```

Comment:
Doesn't seem like these need to be mutually exclusive, right?  The command doesn't give any feedback like that, so might as well allow both.  Maybe ```
if not outline and not dump:
    ... actual build ...
else:
    if outline:
        ... outline ...
    if dump:
        ... dump ...
```

?